packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -0,0 +1,234 @@
+var fs = require('fs');
+var path = require('path');
+var childProcess = require('child_process');
+var flowBinPath = require('flow-bin');
+const flowTypedPath = path.join(__dirname, 'node_modules', '.bin', 'flow-typed');
```

Comment:
We could do that but there is unfortunately no documented JS API on `flow-typed`, I prefer using the "known" CLI API even if that means launching a subprocess.